libvmi/accessors.c

```diff
@@ -335,10 +335,10 @@ GSList* vmi_get_va_pages(vmi_instance_t vmi, addr_t dtb) {
 
 addr_t vmi_pagetable_lookup (vmi_instance_t vmi, addr_t dtb, addr_t vaddr)
```

Comment:
@awsaba I'm not against changing the external API, especially for a case such as this given that it helps to remove ambiguity.  +1 from me, but will defer to @bdpayne